setup.py

```diff
@@ -10,7 +10,7 @@
     install_requires=['thrift==0.10.0',
                       'jsonpickle',
                       'six',
-                      'basictracer>=2.2,<2.3'],
+                      'basictracer==3.0.0rc1'],
```

Comment:
Should we commit this to master or put this in a release branch?

Reply:
I don't think we shouldn't release this to master, but to a release branch.

For `opentracing-python`, we keep a branch for the next version (such as `v2.2.0`) where we keep the upstream work and do release candidates from it. Maybe we could use that style or a similar one?

Reply:
Sounds great to me!